File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get hight from user
    int hight;
    do
    {
        hight = get_int("Hight: ");
    }
    while (hight < 1 || hight > 8);

    // Print out blocks
    for (int i = 0; i < hight; i++)
    {
        for (int o = hight - 1; o > i; o--)
        {
            // Print space
            printf(" ");
        }
        for (int j = -1; j < i; j++)
        {
            // Print hashtag
            printf("#");
        }
        //Print a new line
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


# Get hight from user
hight = get_int("Hight: ")

while hight < 1 or hight > 8:
    hight = int(input("Hight: "))


# Print out blocks
for i in range(hight):
    for k in range(hight - i - 1):
        # Print space
        print(" ", end="")
    for j in range(i + 1):
        # Print hashtag
        print("#", end="")
    print()
